packages/data-broker/src/__mocks__/utils.js

```diff
@@ -0,0 +1,8 @@
+/**
```

Comment:
The existence of this file in __mocks__ tells jest to auto mock the utils package. The alternative of doing it in the test file itself doesn't work due to a limitation of jest.